### PR TITLE
Chore/project docs update

### PR DESCRIPTION
**Pull request by thescientist13**
## Related Issue
N / A

## Summary of Changes
Updated CONTRIBUTING.md with
1. _Project Management_ section covering Discussions and Project boards
1. Added _Release Management_ section
1. Did a little re-structuring of headings

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -50,16 +48,34 @@ Lifeycles include handling:
 - prerendering a compilation for production
 - fetching external (content) data sources
 
+## Project Management
+
+We take advantage of quite a few features on GitHub to assist in tracking issues, bugs, ideas and more for the project.  We feel that being organized not only helps the team in planning out priorities and ownership, it's also a great way to add visisbility and transparency to those following the project.
+
+### Project Boardss
```
Comment (hutchgrant) [inline .github/CONTRIBUTING.md:55]: Boards

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 ## Welcome!
 We're excited for your interest in Greenwood, and maybe even your contribution!
 
-> _We encourage all contributors to have first read about the project's vision and motivation's on the website's [About page](https://www.greenwoodjs.io/about/).  Greenwood is opinionated in the sense that is designed to support development for the web platform and deliver a first class developer experience tailored around that, so that anyone can create a modern and performant website (or webapp, if you prefer). So if that page is the "why", this page is the "how"._
+> _We encourage all contributors to have first read about the project's vision and motivation's on the website's [About page](https://www.greenwoodjs.io/about/).  Greenwood is opinionated in the sense that it is designed to support development for the web platform and deliver a first class developer experience tailored around that, so that anyone can create a modern and performant website (or webapp, if you prefer) knowing just standard web fundamentals. So if that page is the "why", this page is the "how"._
```
Comment (hutchgrant) [inline .github/CONTRIBUTING.md:6]: it's ? not it is. Contraction or no?

Comment (hutchgrant) [reply .github/CONTRIBUTING.md:6]: "for the web platform" sounds weird maybe just "for the web ? "

Comment (hutchgrant) [reply .github/CONTRIBUTING.md:6]: That ^^ sentence kinda run-on

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -50,16 +48,34 @@ Lifeycles include handling:
 - prerendering a compilation for production
 - fetching external (content) data sources
 
+## Project Management
+
+We take advantage of quite a few features on GitHub to assist in tracking issues, bugs, ideas and more for the project.  We feel that being organized not only helps the team in planning out priorities and ownership, it's also a great way to add visisbility and transparency to those following the project.
+
+### Project Boardss
+
+Our [sequentially named project boards](https://github.com/ProjectEvergreen/greenwood/projects) help us organize work in quartery buckets with a small handful of "top line" goals and objectives we would like to focus on for the upcoming time box.  It also serves as a catch-all for the usual work and bug fixes that happen throughout general maintenance of the project and can also yield good oppourtunities for those interested in contributing to see what we would appreciate help with the most.
```
Comment (hutchgrant) [inline .github/CONTRIBUTING.md:57]: quarterly ? I think you mean?

oppourtunities  ?  Opportunities

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -50,16 +48,34 @@ Lifeycles include handling:
...
+
+### Discussions
+
+We believe good collaboration starts with good communication.  As with most of the open source community, Greenwood is a 100% volunteer project and we understand the importance of respecting everyones [time and expectations](https://www.jason.af/setting-expectations).  Although we don't mind issues being made, unless the issue is clearly actionable and falls in-line with the motiviations and trajectory of the project, then feel free to go ahead an open a [Discussion](https://github.com/ProjectEvergreen/greenwood/discussions) first.
+
+We encourage discussions as we believe it is better to hash out technical discussions and proposals ahead of time since coding and reviewing PRs is very time consuming and as maintainer's, we want to make sure everyone gets the time they are deserved for contributing and this helps us plan our time in advance to best ensure a smooth flow of contributions through the project.
```
Comment (hutchgrant) [inline .github/CONTRIBUTING.md:63]: "it is" can probably go with contraction instead.  Also this is a single sentence and it's long.

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -50,16 +48,34 @@ Lifeycles include handling:
...
+
+> _Put another way, we like to think of this approach as **measure twice, cut once**._
+
+### Issues
+We like to reserve issues for features and requests that are more or less "shovel" ready.  This could include prior discussions with the team or coming over from an existing Disussion.
```
Comment (hutchgrant) [inline .github/CONTRIBUTING.md:68]: capital D on discussion not needed

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -150,7 +166,11 @@ Here are some thigns to keep in mind while writing your tests, due to the asynch
 - All usages of `JSDOM` should be wrapped in `async`
 - Avoid arrow functions in mocha tests (e.g. `() => `) as this [can cause unexpected behaviors.](https://mochajs.org/#arrow-functions).  Just use `function` instead.
 
-## Internet Explorer
+## Suppplemental Infomration
```
Comment (hutchgrant) [inline .github/CONTRIBUTING.md:169]: information

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -210,4 +230,16 @@ When you're done with the container:
 1. Exit the container: `$ exit`
 1. Destroy the container: `$ docker rm greenwood`
 
-Note: If you have issues running tests due to timeouts, you can increase the setting in [package.json](https://github.com/ProjectEvergreen/greenwood/blob/master/package.json#L23)
\ No newline at end of file
+Note: If you have issues running tests due to timeouts, you can increase the setting in [package.json](https://github.com/ProjectEvergreen/greenwood/blob/master/package.json#L23)
+
+## Release Management
+
+Lerna is used to manage the publishing of packages within the workspace.  Assuming your are logged into **npm** locally and have 2FA access to publish, the command to run is
```
Comment (hutchgrant) [inline .github/CONTRIBUTING.md:237]: "your are"  - you're

File: .github/ISSUE_TEMPLATE.md [hunk 8]
```diff
@@ -6,11 +6,11 @@
 - [ ] Other (please clarify below)
 
 ## Summary
-<!-- Brief summary of the issue. -->
+<!-- Brief summary of the issue like steps to reproduce for a bug, or motivation and API design if proposing a new feature. -->
 
 ## Details
 <!-- 
-Include relevant code samples, example snippets, benefits, complexity, etc.
+Include relevant code samples, example snippets, benefits, complexity, etc.  For feature requests, please make sure to provide an actionable outcome.  If you're unsure what qualifies, please consider start6ing a Discussion first, we can always transfer it later :).
```
Comment (hutchgrant) [inline .github/ISSUE_TEMPLATE.md:13]: "start6ing"  and "Discussion"

**Review by hutchgrant — CHANGES_REQUESTED** [review]
spelling errors, typos, grammer, and a few run-ons.

**Comment by thescientist13** [comment]
Think I got em all @hutchgrant , thanks for the keen review.  🔍 👀

**Pull request merged**